code/game/machinery/machinery.dm

```diff
@@ -116,8 +116,8 @@ Class Procs:
 	var/panel_open = 0
 	var/area/myArea
 	var/interact_offline = 0 // Can the machine be interacted with while de-powered.
-	var/use_log = list()
-	var/list/settagwhitelist = list()//WHITELIST OF VARIABLES THAT THE set_tag HREF CAN MODIFY, DON'T PUT SHIT YOU DON'T NEED ON HERE, AND IF YOU'RE GONNA USE set_tag (format_tag() proc), ADD TO THIS LIST.
+	var/list/use_log // Init this list if you wish to add logging to your machine - currently only viewable in VV
```

Comment:
This is actually used by the particle accelerator

Reply:
I'm aware it is but I forgot to initialize the list there 🤦‍♀️ It's fixed in next commit.